site/app/models/RainbowCustomization.php

```diff
@@ -52,8 +69,6 @@ public function __construct(Core $main_core) {
         $this->has_error = "false";
         $this->error_messages = [];
 
-        $this->sections = (object) [];
```

Comment:
Is this no longer used  ?

Reply:
I don't know if it ever was, but it does not seem to be used now.